lib/src/data/irma_preferences.dart

```diff
@@ -25,7 +25,6 @@ class IrmaPreferences {
         _reportErrors = preferences.getBool(_reportErrorsKey, defaultValue: false),
         _startQRScan = preferences.getBool(_startQRScanKey, defaultValue: false),
         _showDisclosureDialog = preferences.getBool(_showDisclosureDialogKey, defaultValue: true),
-        _developerModePrefVisible = preferences.getBool(_developerModePrefVisibleKey, defaultValue: false),
```

Comment:
Maybe nice to remove the preference too then, like we do for `_showNameChangeNotificationKey`.